app/views/layouts/application.html.erb

```diff
@@ -60,6 +60,8 @@
         </p>
       </div>
 
+      <%= yield :back_button %>
```

Comment:
Will this duplicate a back button for the form steps?

Reply:
I think in the form steps we should have something like

```ruby
<% content_for(:back_button) %>
  <%= render GovukComponent::BackLink.new(text: 'Dashboard',href: dashboards_matchmaker_path) %>
<% end %>
```

And it will always put it in the right place and we can position it only once in the layout.